src/gui/entry/EntryModel.h

```diff
@@ -34,20 +34,21 @@ class EntryModel : public QAbstractTableModel
     enum ModelColumn
     {
         ParentGroup = 0,
-        Title = 1,
-        Username = 2,
-        Password = 3,
-        Url = 4,
-        Notes = 5,
-        Expires = 6,
-        Created = 7,
-        Modified = 8,
-        Accessed = 9,
-        Paperclip = 10,
-        Attachments = 11,
-        Totp = 12,
-        Size = 13,
-        PasswordStrength = 14
+        Color = 1,
```

Comment:
This should be at the bottom not the top

Reply:
So it should be ID 15 instead of 1? 

I assumed the default order was defined here, and i put it where i thought best of, logically.